src/main/steps/eligibility/domicile/content.test.ts

```diff
@@ -0,0 +1,85 @@
+import { YesNoNotsure } from '../../../app/case/definition';
+import { FormContent, FormFields, FormOptions } from '../../../app/form/Form';
+import { isFieldFilledIn } from '../../../app/form/validation';
+import { CommonContent, generatePageContent } from '../../common/common.content';
+
+import { generateContent } from './content';
+
+jest.mock('../../../app/form/validation');
+
+const enContent = {
+  section: 'Eligibility to apply to adopt',
+  label:
+    'Is the UK, Channel Islands or Isle of Man the main country of residence (domicile) for you and the other applicant if relevant?',
+  domicileNo: 'You cannot apply to adopt a child unless you have a permanent home here.',
+  moreInfo: 'More about adoption',
+  errors: {
+    domicileEligible: {
+      required: 'Please answer the question',
+    },
+  },
+};
+
+const cyContent = {
+  section: 'Eligibility to apply to adopt (in Welsh)',
+  label:
+    'Is the UK, Channel Islands or Isle of Man the main country of residence (domicile) for you and the other applicant if relevant? (in welsh)',
+  domicileNo: 'You cannot apply to adopt a child unless you have a permanent home here. (in welsh)',
+  moreInfo: 'More about adoption (in welsh)',
+  errors: {
+    domicileEligible: {
+      required: 'Please answer the question (in welsh)',
+    },
+  },
+};
+
+/* eslint-disable @typescript-eslint/ban-types, @typescript-eslint/no-explicit-any */
+describe('eligibility > domicile > content', () => {
+  const commonContent = generatePageContent({
+    language: 'en',
+    userCase: {},
+  }) as CommonContent;
+
+  let generatedContent;
+  beforeEach(() => {
+    generatedContent = generateContent(commonContent);
+  });
+
+  test('should return correct english content', () => {
+    expect(generatedContent.section).toEqual(enContent.section);
+    expect(generatedContent.label).toEqual(enContent.label);
+    expect(generatedContent.domicileNo).toEqual(enContent.domicileNo);
+    expect(generatedContent.moreInfo).toEqual(enContent.moreInfo);
+    expect(generatedContent.errors).toEqual(enContent.errors);
```

Comment:
can be simplified using languageAssertions
example https://github.com/hmcts/adoption-web/blob/05ddc4ab44fa2e4bee1632469dc6df594ee7ca22/src/main/steps/sibling/placement-order-check-your-answers/content.test.ts#L51